### DISCUSSION TITLE
Wireframe varianter av komponenter i Figma

### DISCUSSION THREAD

**Discussion by Murstam** [fremtind/jokul#2693] [Annet]
Hei! Tror denne har vært diskutert før, men husker ikke helt hva vi landa på. Kan det være en idé å lage wireframe varianter av komponentene i Figma? Fordelene er når man er på et tidlig stadie i skissene og ikke ønsker fokus på visuelle detaljer eller for å synliggjøre at dette ikke er endelige skisser. Hva tenker folk?

**Comment by Slaine2** [comment]
Ja, jeg tror det kan gjøre brainstorming og planlegging enklere, så det ville vært raskere. Kan det også være fornuftig å tenke på komplette UI-elementer og -seksjoner?

**Reply by Murstam** [reply]
Ja, UI-elementer og -seksjoner for en del standard oppsett tror jeg er en god idé 👍

**Comment by piofinn** [comment]
Helt enig! Tror det kunne hjulpet mye i diskusjoner rundt skisser på et tidlig stadie, så man ikke ender opp med å diskutere detaljer for tidlig 😊

**Comment by wkillerud** [comment]
Vi diskuterte dette på forumet i går, og selv om det er et ønske er det ikke høyt på prioriteringslista enn så lenge.

**Comment by kristineSorensen** [comment]
Fra Ixd-forum. VI har diskutert dette tidligere og lastet da ned et bibliotek som var ok. https://www.figma.com/file/zkD0hEQzsE7Pjpep5FSkUN/Tr%C3%A5dkisser?node-id=0%3A1 

V kan jo iterere på dette biblioteket.

**Reply by efatland** [reply]
Biblioteket heter "Trådskisser" i Figma. Det har allerede fått noen nye komponenter inspirert av Jøkul, som Fremtind-logoen og varsel/infomeldinger. Tanken er at man skal ha det samme vokabularet med dette biblioteket som man har med Jøkul.
